src/features/expenses/AddExpense/AddPayers.tsx

```diff
@@ -0,0 +1,154 @@
+import { useCallback, useEffect, useMemo, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+
+import { TextUnderline } from "components";
+import { Column, PageWrapper, Row } from "components/Containers";
+import Header from "components/Header";
+import { ReactComponent as BackIconSVG } from 'assets/images/back-icon.svg';
+import { ReactComponent as ArrowRightSVG } from 'assets/images/arrow-right.svg';
+import { ReactComponent as ArrowRightDisabledSVG } from 'assets/images/arrow-right-disabled.svg';
+import { BackgroundFiller, ContentWrapper, HalfCircleBackground, AddPayersWrapper, NextStepButton, AddPayersText, CurrencyText, AddPayersInfo } from "./AddPayers.styled";
+import BottomNavigation from "components/BottomNavigation";
+import currencies from "constants/currencies";
+import { ICreateTransaction } from "../types";
+import { newTransactionSelector, saveTransaction } from "../slice";
+import { CreateRouteString, ExpenseRouteMode, composeExpenseRoute } from "constants/routes";
+import { getCurrencyWithSymbolString } from "helpers/currencyHelper";
+import UserListItem from "components/UserListItem";
+import { useGetTransactionGroupByIdQuery } from "features/groups/api";
+import { useModal } from "providers/ModalProvider";
+import ChangeUserMoneyModal from "../ChangeUserMoneyModal";
+import { IUser } from "types/userTypes";
+
+const getUserAmountComponent = (user: IUser, currencyCode: string, userAmountMap: Record<string, number>) => {
+  const amount = userAmountMap[user.id] > 0 ? userAmountMap[user.id] : 0;
+  return <CurrencyText>{getCurrencyWithSymbolString(amount, currencyCode)}</CurrencyText>;
+};
+
+const AddPayers = () => {
+  const { t } = useTranslation('expenses');
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const modalContext = useModal();
+
+  const transaction = useSelector(newTransactionSelector);
+  const group = useGetTransactionGroupByIdQuery(transaction?.groupId || '');
```

Comment:
What should be as a 'group' when empty ID provided?

Reply:
I fixed the case when me are missing transaction from selector, please see: c0490830501697a7465016d0864d5e7afa56ee71.
If transaction is present then we also have groupId.